views/javascript.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener('DOMContentLoaded',function(event) {
+  var calculatorNumbers = document.getElementById('calculator-numbers')
+  calculatorNumbers.addEventListener('click', eventLogger)
+})
+
+// function sum() is a callback that is waiting for the return of rightSideOfOperation()
+// rightSideOfOperation() starts when + or * is == value
+
+function displayIt(value) {
+  var currentValue = document.getElementsByTagName('P')[0].innerText
+    if(currentValue[0] == 0) {
+      document.getElementsByTagName('P')[0].textContent = value
+    }else{
+      document.getElementsByTagName('P')[0].textContent = currentValue + value
+    }
+    if(value =='+') {
+      var operation = '+'
+      holdOperator(value, currentValue, operation)
+    }
+    setToZero(value,currentValue)
+    return currentValue
+}
+
+function holdOperator(value, currentValue, operation) {
+
+  console.log('VALUE', value)
+  console.log('currentValue', currentValue)
+  document.getElementsByTagName('P')[0].innerText = operation
+  if(currentValue[0] == '+') {
+    document.getElementsByTagName('P')[0].innerText = currentValue.substr(1)
+    
+  }
+
+
+  return operation
+}
+
+
+function rightSideOfOperation(currentValue, value, operation) {
```

Comment:
It was where I stoped with the goal. The logic behind this concept as to separate the entries from the user as 'left side of operator' and 'right side of operator' to collect the numbers to add, subtract, multiply, or divide...